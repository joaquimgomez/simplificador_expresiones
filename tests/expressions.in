PREFIXA  and a or b not and T a -> INFIXA
POSTFIXA a T and not b or a and -> PREFIXA
PREFIXA  and or or F a b or a and b b -> INFIXA
POSTFIXA b b and a or b F a or or and -> POSTFIXA
POSTFIXA 3 5 + x == 3 4 - x * 8 != not or -> POSTFIXA
POSTFIXA 12 9 - 23 5 + -2 * / -> INFIXA
PREFIXA - + + x 3 * 2 y + * 2 y + 3 x -> INFIXA 
POSTFIXA x 12 + 12 x + / -> INFIXA 
