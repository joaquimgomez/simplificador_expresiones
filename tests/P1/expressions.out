(a and (b or (not (T and a))))
and or not and a T b a 
(((F or a) or b) and (a or (b and b)))
b b and a or b F a or or and 
3 5 + x == 3 4 - x * 8 != not or 
((12 - 9) / ((23 + 5) * -2))
(((x + 3) + (2 * y)) - ((2 * y) + (3 + x)))
((x + 12) / (12 + x))
